client/src/components/Videos.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Rating from "./Rating";
+
+function Videos(props) {
+  // Split after '=' get second array allows to get embed!
+
+  let src = "https://www.youtube.com/embed/" + props.data.url.split("=")[1];
+
+  return (
+    <div>
+      <div className="container">
+        <div
+          className="card mb-4 shadow-sm"
+          style={{
+            background: "black",
+            border: "#D43C31 solid",
+            margin: "2px",
+            padding: "2px",
+          }}
```

Comment:
Really liked the idea of inline component styling here. Can you please explain me why have you chosen to do it like this?(There is nothing wrong with it, just want to know the reason for it.)

Reply:
It was from react boostrap :smile: I wanted to make easy.